dashboard/app/views/admin_reports/funometer.html.haml

```diff
@@ -1,100 +1,77 @@
 - content_for :head do
   %script{src:'https://www.google.com/jsapi'}
 
-%h1
-  Fun-O-Meter Stats
+- if @stats.present?
+  %h1
+    Fun-O-Meter Stats
 
-%h2
-  Overall Percentage:
-  %span= number_to_percentage(@overall_percentage, precision: 2)
+  %h2
+    Overall Percentage:
+    %span= number_to_percentage(@stats['overall'], precision: 2)
 
-%br
+  %br
 
-= render 'funometer_percentages_by_day', :percentages_by_day => percentages_by_day
+  -# Renable the graphs after reconciling the different object types.
```

Comment:
Not sure. If so, note that we'd need to save the results for every script and every level.

The impetus for changing this dashboard was it was timing out. The other dashboards are (not) yet timing out, so the impetus may or may not be there. Thoughts?